doc/source/examples/other/parmed_sim.ipynb

```diff
@@ -31,7 +31,15 @@
      "start_time": "2020-12-27T11:14:14.063236Z"
```

Comment:
<p>probably will want a filterwarning for this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>